server/sonar-server/src/main/java/org/sonar/server/rule/RuleService.java

```diff
@@ -93,7 +68,11 @@ public RuleQuery newRuleQuery() {
    */
   public Set<String> listTags(@Nullable String query, int size) {
     /** using combined ALL_TAGS field of ES until ES update that has multiTerms aggregation */
-    return index.terms(RuleNormalizer.RuleField.ALL_TAGS.field(), query, size);
+    return index.terms(RuleIndexDefinition.FIELD_RULE_ALL_TAGS, query, size);
+  }
+
+  public SearchIdResult<RuleKey> search(RuleQuery query, SearchOptions options) {
+     return index.search(query, options);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)